docs/contributor_guide/contributing_documentation.rst

```diff
@@ -16,7 +16,9 @@ To contribute, make sure to install sphinx and its add-ons by running
     python scripts/install_requirements.py --pinned False
 
 in the repository root directory.
-You may also need to `install Pandoc <https://pandoc.org/installing.html>`_.
+You may also need to `install Pandoc <https://pandoc.org/installing.html>`_, and
+if you are a MacOS user in some cases also `Lightgbm
+<https://lightgbm.readthedocs.io/en/stable/Installation-Guide.html>`_.
```

Comment:
why is `install_requirements.py` not handling this then? I think the fix should go there?

Reply:
The change reverted in e391f0fd13c3ba6e3890371b19621e7b1d475621.